### PR TITLE
feature: add tint color to sort arrow

### PR DESCRIPTION
**Pull request by altagir**
add tint color to sort arrow (sortArrowTintColor in DataTableConfiguration)

### CODE REVIEW

File: Example/DemoSwiftDataTables/DataTableWithDataSetViewController.swift [hunk 1]
```diff
@@ -23,6 +23,7 @@ class DataTableWithDataSetViewController: UIViewController {
         
         var options = DataTableConfiguration()
         options.shouldContentWidthScaleToFillFrame = false
+        options.sortArrowTintColor = UIColor.orange
```
Comment (pavankataria) [inline Example/DemoSwiftDataTables/DataTableWithDataSetViewController.swift:26]: Is there a reason why this is orange as opposed to the original blue color?

Comment (pavankataria) [reply Example/DemoSwiftDataTables/DataTableWithDataSetViewController.swift:26]: Sorry it took so long, I forgot to submit the review. It was just in a forever pending state!

Comment (altagir) [reply Example/DemoSwiftDataTables/DataTableWithDataSetViewController.swift:26]: I did it intentionally in one of the 2 samples, to clearly show to whoever runs the demo, that it was customizable.

and with color different than the default

**Comment by altagir** [comment]
I am unsure I will push you pull request in the future. 
It took 26 days just to get an answer.
I will continue on my fork

**Comment by pavankataria** [comment]
Sorry it took so long. There were some issues at work which forced me to change jobs. 
I've taken another look now and this is great!
Thanks for the contribution!

Will merge this in.

**Pull request merged**